src/test/java/org/folio/linked/data/e2e/ResourceControllerIT.java

```diff
@@ -472,8 +472,8 @@ void deleteResourceById_shouldDeleteRootWorkAndRootEdges() throws Exception {
     // given
     var existed = resourceTestService.saveGraph(getSampleWork(getSampleInstanceResource(null, null)));
     assertThat(resourceTestService.findById(existed.getId())).isPresent();
-    assertThat(resourceTestService.countResources()).isEqualTo(50);
-    assertThat(resourceTestService.countEdges()).isEqualTo(52);
+    assertThat(resourceTestService.countResources()).isEqualTo(53);
```

Comment:
minor, thoughts outloud

@pkjacob  regarding our discussion about complicated tests.
You can see this problem in our link-data module, too.
For example, Integers like 53, 55, and others can't show the real situation; they only stop the process and hide problems after it.
We need to change the situation in the future and change our approach from "all-in-one" test to separate test cases